app/src/pages/CreateCompetition/CreateCompetition.jsx

```diff
@@ -224,7 +228,15 @@ function CreateCompetition() {
           )}
         </div>
         <div className="form-row form-actions">
-          <Button text="Confirm" onClick={onSubmit} loading={isSubmitting} />
+          <Button
+            text="Confirm"
+            onClick={
+              (!groupCompetition && participants.length === 0) || (groupCompetition && !selectedGroup)
```

Comment:
This could look cleaner if you declared a "isEmpty" variable before the return statement, and then used it in this validation.

`isEmpty ? showEmptyCompetitionModal : onSubmit`